packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -93,13 +93,22 @@ module.exports = {
     }
   },
 
-  // Truffle DB is currently disabled by default; to enable it, change enabled: false to enabled: true
+  // Truffle DB is currently disabled by default; to enable it, change enabled:
+  // false to enabled: true. The default storage location can also be
+  // overridden by specifying the adapter settings, as shown in the commented code below.
   //
-  // Note: if you migrated your contracts prior to enabling this field in your Truffle project and want
-  // those previously migrated contracts available in the .db directory, you will need to run the following:
+  // Note: your previously migrated contracts will not be available in
+  // truffle-db. We recommnd you backup your artifacts before utilizing Truffle
```

Comment:
```suggestion
  // truffle-db. We recommend you backup your artifacts before utilizing Truffle
```

Reply:
Changed the text to remove ambiguity. See latest changes